<?xml version="1.0"?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from mmrobot.urdf.xacro             | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="albert">
  <link name="world"/>
  <joint name="boxer_joint_world" type="fixed">
    <parent link="world"/>
    <child link="odom"/>
    <origin rpy="0 0 0" xyz="0 0 0"/>
  </joint>
  <link name="odom"/>
  <link name="world_base_link_x"/>
  <link name="world_base_link_y"/>
  <link name="base_link_ori"/>
  <link name="base_link"/>
  <joint name="world_base_x" type="prismatic">
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <parent link="odom"/>
    <child link="world_base_link_x"/>
    <axis xyz="1 0 0"/>
    <limit effort="100" lower="-15" upper="15" velocity="5"/>
  </joint>
  <joint name="world_base_y" type="prismatic">
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <parent link="world_base_link_x"/>
    <child link="world_base_link_y"/>
    <axis xyz="0 1 0"/>
    <limit effort="100" lower="-15" upper="15" velocity="5"/>
  </joint>
  <joint name="world_base_theta" type="revolute">
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <parent link="world_base_link_y"/>
    <child link="base_link_ori"/>
    <axis xyz="0 0 1"/>
    <limit effort="100" lower="-15" upper="15" velocity="5"/>
  </joint>
  <joint name="base_add_orientation" type="fixed">
    <origin rpy="0 0 1.57079" xyz="0.0 0 0"/>
    <parent link="base_link_ori"/>
    <child link="base_link"/>
  </joint>
  <link name="ee_link"/>
  <joint name="ee_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 -0.4 0"/>
    <parent link="base_link"/>
    <child link="ee_link"/>
  </joint>
</robot>

